### PR TITLE
docs: add quick start to the readme

### PR DESCRIPTION
**Pull request by mert-kirpici**

### CODE REVIEW

**Comment by mert-kirpici** [comment]
description section in the snap store will be updated after the reviews

**Review by aieri — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -51,7 +57,25 @@ List of plugins:
...
+sudo snap install tempest --edge
+
+# Have the cloud credentials in the environment
+source novarc
+
+# Initialize a tempest workspace directory called cloud-01
+tempest init cloud-01 && cd cloud-01
+
+# Generate the tempest configuration specific to the cloud
+discover-tempest-config
```
Comment (agileshaw) [inline README.md:73]: We should wait for this default alias to take effect before merging the PR.

Comment (mert-kirpici) [reply README.md:73]: it took effect. resolving this conversation.

Comment (rgildein) [reply README.md:73]: Adding link to the [request](https://forum.snapcraft.io/t/auto-connect-alias-request-for-tempest/36422).

**Review by rgildein — APPROVED** [review]
LGTM

**Comment by mert-kirpici** [comment]
Non development related section of the readme now live in the snap store as the description for the [tempest snap](https://snapcraft.io/tempest). Merging this PR. thanks for the reviews.

**Pull request merged**